src/Verify.ApprovalTestsTransition/Verify.ApprovalTestsTransition.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT'">net472;net48</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworks);net6.0;net7.0;net8.0</TargetFrameworks>
```

Comment:
nit: You don't need the `'$(OS)' == 'Windows_NT'` conditions anymore as the .NET Framework automatically pulls in the .NET Framework targeting pack when building on Windows/Unix.

You could just collapse these to: `<TargetFrameworks>net472;net48;net6.0;net7.0;net8.0</TargetFrameworks>`.

Reply:
@ViktorHofer awesome. thanks for the hint